src/main/webapp/resources/sass/app.scss

```diff
@@ -385,3 +385,7 @@ ol.continue li:before {
   padding-top: 5px;
   padding-bottom: 5px;
 }
+
+label {
+  font-weight: normal !important;
```

Comment:
So this will overwrite all labels in the project, is that what you intended?

Reply:
Not at all. Fixed ina 05ed339a1442b2b20b020099d6aab160bdbb348